### ISSUE TITLE
Target profile page rewrite: Create "Cancer Biomarkers" summary widget

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#247]

**Comment by andrewhercules** [comment]
### Use case(s)

1. As a user, I want to know if a biomarker associated with a gene influences the responsiveness of a given drug when treating a specific type of cancer

### Summary widget (showing different states depending on data)

![cancer-biomarkers](https://user-images.githubusercontent.com/7490258/49018110-72de1680-f182-11e8-8dd0-5ceb91ef0358.png)

**[Full size version](https://drive.google.com/open?id=1IlSSzQG7y7J0QFtC6H34YTVPrxvaqjn_)**
 
### Key design and interaction notes

- If we have data for a given gene, please make the widget icon, widget border, and widget text dark grey (#5A5F5F) 
- If we do not have data for a given gene, please make widget icon, widget border, and widget text light grey (#E2DFDF) 

### Design assets

1. The widget icon is available on the [OT Core Platform Team Google Drive](https://drive.google.com/drive/folders/1KOgvjVQYhIS1099lhhdPvfEwXnR0whVk)
2. Colour codes are available on the [OT Core Platform Team Google Drive](https://drive.google.com/open?id=1d0JDmCznInsANZoQC77gadTf9jO0UDOo)

**Comment by peatroot** [comment]
Proposed schema:
```
type TargetSummaryCancerBiomarkers {
  hasCancerBiomarkers: Boolean!
  cancerBiomarkerCount: Int!
  diseaseCount: Int!
  drugCount: Int!
}
```

**Comment by peatroot** [comment]
Now looks like this.

**Comment by andrewhercules** [comment]
The layout of this widget may need to be updated once the max-width container is implemented - see issue #366 for further details
